package.json

```diff
@@ -1,14 +1,19 @@
 {
   "name": "@asyncapi/avro-schema-parser",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
bot changes version, please remove this change from here and `package-lock.json`